valhalla/sif/edgelabel.h

```diff
@@ -26,7 +26,11 @@ class EdgeLabel {
   /**
    * Default constructor.
    */
-  EdgeLabel() {
+  EdgeLabel()
+      : predecessor_(0), path_distance_(0), restrictions_(0), edgeid_(0), opp_index_(0),
```

Comment:
i think predecessor should be kInvalidLabel and the graphids should be invalid ones